src/components/AddFriend/FriendRequests.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from "react";
+import { socket } from "../../pages/CommonLayout/CommonLayout";
+import { ListStyles } from "../FriendList/FriendList.styles";
+import DoneIcon from "@mui/icons-material/Done";
+import CloseIcon from "@mui/icons-material/Close";
+import { IconButton, Box, Avatar, Typography } from "@mui/material";
+import { useSelector } from "react-redux";
+import { allRequests } from "../../api/services/FriendRequestApi";
+
+const FriendRequests = ({ setshowAlertToast }) => {
+  const { classes } = ListStyles();
+
+  const [friendRequests, setFriendRequests] = useState([]);
+
+  const userId = useSelector((state) => {
+    return state?.userDataReducer[0]?.data?.user.id;
+  });
+
+  useEffect(() => {
+    const run = async () => {
+      const requests = await allRequests(userId);
+      setFriendRequests(requests);
+    };
+    run();
+  }, [userId]);
+
+  socket.on("followRequest", (data) => {
+    setFriendRequests([...friendRequests, data]);
+  });
+
+  const acceptRequest = (sender, receiver, id) => {
+    socket.emit("requestAccepted", {
+      followerUserId: sender,
+      followedUserId: receiver,
+      status: "Accepted",
+    });
+    const accept = friendRequests.filter((singleRequest) =>
+      singleRequest?.id
+        ? singleRequest.id
+        : singleRequest?.newRelationship.id !== id
+    );
+    setFriendRequests(accept);
+    setshowAlertToast({
+      visiblity: true,
+      message: "Friend request accepted !",
```

Comment:
make a constants file and add these string messages to that file.

Reply:
"Friend request accepted !" and "success" these messages can be written in constant file